.gitlab-ci.yml

```
@@ -0,0 +1,21 @@
+image: nikolaik/python-nodejs:python3.8-nodejs12
```

```suggestion
image: kiwicom/sls
```

I'd rather use our own image for this :)
It doesn't have python 3.8 though, but we don't need it at the moment

---

Here is the dockerfile for that image: https://github.com/kiwicom/dockerfiles/blob/master/sls/Dockerfile
This is actually the image we use internally for sls deployments